### PR TITLE
Multiple fix template and license

### PR DESCRIPTION
**Pull request by douph1**
Cummulated goal of this PR:

- fix concurent build issue
- merge Wix template from Hotspot/OpenJ9/Dragonwell into same file
- #162 Do not require users to explicitly accept the terms of the GPLv2+CE
- #300 move CurrentVersion test to CurrentVersion reg key only 
- build installer jdk11 with github workflow

### CODE REVIEW

File: wix/Includes/OpenJDK.Variables.wxi.template [hunk 1]
```diff
@@ -56,23 +56,40 @@
...
+    <?define license_shown="1" ?>
+  <?endif?>
+
+  <?if $(var.JVM)="openj9"?>
+    <?define DllPath="bin\server\jvm.dll" ?>
+  <?elseif $(var.JVM)="hotspot" ?>
+    <!-- cf https://github.com/AdoptOpenJDK/openjdk-installer/issues/137 -->
+    <?if $(env.PRODUCT_CATEGORY)="jre"?>
+      <?if $(var.ProductMajorVersion)>=9?>
+          <!-- Prefere Server JVM in Java 9+ x64 -->
```
Comment (tellison) [inline wix/Includes/OpenJDK.Variables.wxi.template:76]:
```suggestion
          <!-- Prefer Server JVM in Java 9+ x64 -->
```

**Comment by karianna** [comment]
I assume this can wait until post release?

**Comment by tellison** [comment]
It is still a Draft anyway, so I take it that Philippe is still working away.

**Comment by douph1** [comment]
>I assume this can wait until post release?
yes, need some more checking/testing

**Review by gdams — APPROVED** [review]

**Pull request merged**